Libraries/PushNotificationIOS/PushNotificationIOS.js

```diff
@@ -55,9 +55,35 @@ class PushNotificationIOS {
    *
    * - `fireDate` : The date and time when the system should deliver the notification.
    * - `alertBody` : The message displayed in the notification alert.
+   * - `repeatInterval` : The interval to repeat as a string.  Possible values: `minute`, `hour`, `day`, `week`, `month`, `year`.
    *
    */
   static scheduleLocalNotification(details: Object) {
+    if (details) {
+      switch (details.repeatInterval) {
+          case 'year':
+              details.repeatInterval = 4; // NSCalendarUnit.CalendarUnitYear
+              break;
```

Comment:
It would be nice to have flow types here for the function arguments & maybe instead of a switch case, just define a constant:
```js
const REPEAT_INTERVALS = {
   year: 4,
   month: 8,
   week: 8192,
};

if (details.repeatInterval) {
   details.repeatInterval = REPEAT_INTERVALS[details.repeatInterval] || 0;
}
```